contracts/switchboard/native/ArbitrumL2Switchboard.sol

```diff
@@ -32,35 +30,27 @@ contract ArbitrumL2Switchboard is NativeSwitchboardBase {
     }
 
     /**
-     * @dev Constructor function that sets initial values for the confirmGasLimit, arbsys__, and the NativeSwitchboardBase parent contract.
+     * @dev Constructor function that sets initial values for the arbsys__, and the NativeSwitchboardBase parent contract.
      * @param chainSlug_ A uint32 representing the ID of the L2 network.
-     * @param confirmGasLimit_ A uint256 representing the amount of gas that will be needed to confirm the execution of a transaction on the L2 network.
      * @param initiateGasLimit_ A uint256 representing the amount of gas that will be needed to initiate a transaction on the L2 network.
-     * @param executionOverhead_ A uint256 representing the amount of gas that is needed for a transaction execution overhead on the L2 network.
      * @param owner_ The address that will have the default admin role in the AccessControl parent contract.
      * @param socket_ The address of the Ethereum mainnet Native Meta-Transaction Executor contract.
-     * @param gasPriceOracle_ An IGasPriceOracle contract used to calculate gas prices for transactions.
      */
     constructor(
         uint32 chainSlug_,
-        uint256 confirmGasLimit_,
         uint256 initiateGasLimit_,
-        uint256 executionOverhead_,
         address owner_,
         address socket_,
-        IGasPriceOracle gasPriceOracle_
+        ISignatureVerifier signatureVerifier_
     )
-        AccessControl(owner_)
+        AccessControlExtended(owner_)
```

Comment:
no need of extended for natives as roles will not be chain specific

Reply:
updated